src/rustllvm/RustWrapper.cpp

```diff
@@ -650,22 +574,12 @@ extern "C" LLVMValueRef LLVMDIBuilderInsertDeclareBefore(
     unsigned AddrOpsCount,
     LLVMValueRef DL,
     LLVMValueRef InsertBefore) {
-#if LLVM_VERSION_MINOR >= 6
-#endif
```

Comment:
I propose we add a whole slew of conditionals for every version with all the no-ops we could possibly do ;-)